emulators/qemu/Portfile

```diff
@@ -265,11 +272,6 @@ variant snappy description {Support Snappy compression} {
     depends_lib-append      port:snappy
 }
 
-variant zstd description {Support zstd compression} {
-    configure.args-replace  --disable-zstd --enable-zstd
-    depends_lib-append      port:zstd
-}
-
```

Comment:
I agree that zstd is seeing more widespread use and I am fine with adding zstd as a new default as that is more likely to be already installed anway.

However, I have not encountered any image compressed with lzo yet. Is it really that common? Switching to port:lzo2 seems reasonable, though. But I do not see lzo to be any more important than lzfse or snappy and we keep them as variants.

Reply:
I think that ideally we should have all the compression formats as built-in.  `qemu` no longer builds with `lzo`, but now requires `lzo2`.

I think maybe it makes sense to add `snappy` to this list as well, and that should be fine.

Reply:
I still doubt that the average user will ever need lzfse or snappy with qemu. That adds a few more dependencies, although these are not that big and are only updated infrequently. But okay, to make this simpler, let's include all compression formats by default.